### ISSUE TITLE
Intermittent - Field 'message' on type 'MessageData' is required but missing or empty

### ISSUE THREAD

**Issue by zoomingrocket** [microsoft/ApplicationInsights-Java#2393]
### Expected behavior
Intermittent Warnings for Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:

### Actual behavior
Not sure if this is a true issue and impacts, but reporting to get feedback

### To Reproduce
N/A

#### Sample Application
N/A - Proprietary COTS

### System information
Please provide the following information:
 - SDK Version: 3.3.0 GA, 3.3.1-SNAPSHOT
 - OS type and version: CentOS 7, JDK 11.0.15
 - Application Server type and version (if applicable): COTS
 - Using spring-boot? No

### Logs

```
2022-07-07 22:40:54.972Z INFO  c.m.applicationinsights.agent - ApplicationInsights Java Agent 3.3.0 started successfully (PID 13034)
2022-07-07 22:40:54.972Z INFO  c.m.applicationinsights.agent - Java version: 11.0.15, vendor: Amazon.com Inc., home: /d00/apps/jre
2022-07-08 01:59:41.465Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - Sending telemetry to the ingestion service: 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (future warnings will be aggregated and logged once every 5 minutes)
2022-07-08 02:04:41.468Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 2 times (out of 2918): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (2 times)
2022-07-08 02:16:53.532Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - Sending telemetry to the ingestion service (retry from disk): 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (future warnings will be aggregated and logged once every 5 minutes)
2022-07-08 02:19:41.469Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 1 times (out of 86): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (1 times)
2022-07-08 14:04:41.496Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 1 times (out of 175): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (1 times)
2022-07-08 14:19:41.498Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 4 times (out of 87): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (4 times)
```

```
2022-07-12 17:32:16.100Z INFO  c.m.applicationinsights.agent - ApplicationInsights Java Agent 3.3.1-SNAPSHOT started successfully (PID 21949)
2022-07-12 17:32:16.101Z INFO  c.m.applicationinsights.agent - Java version: 11.0.15, vendor: Amazon.com Inc., home: /d00/apps/jre
2022-07-13 02:20:20.157Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 2 times (out of 85): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (2 times)
2022-07-13 14:05:20.179Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 3 times (out of 145): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (3 times)
2022-07-13 14:20:20.180Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 3 times (out of 79): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (3 times)
2022-07-14 02:00:20.203Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 4 times (out of 95): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (4 times)
2022-07-14 02:20:20.204Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 1 times (out of 72): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (1 times)
 2022-07-14 14:00:20.228Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 2 times (out of 102): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (2 times)
2022-07-14 14:05:20.229Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 1 times (out of 130): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (1 times)
2022-07-14 14:20:20.234Z WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 5 times (out of 85): Sending telemetry to the ingestion service:
 * 109: Field 'message' on type 'MessageData' is required but missing or empty. Expected: string, Actual:
 (5 times)
```

**Comment by trask** [comment]
@zoomingrocket thx for reporting this! it should be fixed now by #2399 and will be included in the next release

**Comment by trask** [comment]
@zoomingrocket thanks for repoting this! it should be resolved in the next release
